Source/Initialization/WarpXInitData.cpp

```diff
@@ -131,21 +131,35 @@ WarpX::InitPML ()
 {
     if (do_pml)
     {
+        amrex::IntVect do_pml_Lo_corrected = do_pml_Lo;
+        // amrex::IntVect do_pml_Hi_corrected = do_pml_Hi;
+
+#ifdef WARPX_DIM_RZ
+        do_pml_Lo_corrected[0] = 0; //no PML along z-axis
```

Comment:
Could you modify the above comment:
``// no PML at r=0, in cylindrical geometry``